config/environment.py

```diff
@@ -51,6 +51,8 @@
     UPLOADS_DEFAULT_DEST = BASE_DIR + '/uploads/'
     UPLOADS_DEFAULT_URL = 'http://localhost:5000/uploads/'
 
+# Enable/disable AB testing
+AB_TESTING = os.environ.get('AB_TESTING') == 'True'
```

Comment:
I'm not sure when Flask loads this, presumably we need to restart the Flask app for this to take effect.

Reply:
I am not sure how the app is deployed but I would assume that it needs to be restarted.

Reply:
yes. It needs to be restarted.